common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -874,6 +876,13 @@ public void run() {
         });
     }
 
+    /**
+     * An interface that is called with a long argument.
+     */
+    protected interface LongCallable {
```

Comment:
@NiteshKant jus a nit but I wonder if we should choose a more meaningful name in this context.